pyeudiw/satosa/backend.py

```diff
@@ -215,12 +215,12 @@ def pre_request_endpoint(self, context: Context, internal_request, **kwargs) ->
 
     def redirect_endpoint(self, context: Context, *args: tuple) -> Redirect | JsonResponse:
```

Comment:
please, rename it to `response_endpoint`

align the init and configuration accordingly